### PR TITLE
fix : #117

### PR DESCRIPTION
**Pull request by praveenpuglia**
installs latest stable google chrome along with it's dependencies and runs test on that.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 node_js:
   - "0.10"
   - "0.12"
 
 before_install:
-  - export CHROME_BIN=chromium-browser
+  - export CHROME_BIN=/usr/bin/google-chrome
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+  - sudo apt-get install -y libappindicator1 fonts-liberation
```
Comment (zdfs) [inline .travis.yml]: Do we need this line?

Comment (zdfs) [inline .travis.yml]: Do we need this line ?

**Comment by zdfs** [comment]
Do we need `fonts-liberation` line?

**Comment by praveenpuglia** [comment]
It did say that in the dependency and failed. Should I check once without it?

**Comment by zdfs** [comment]
If you've already tried, that's good enough.

**Pull request merged**